extensions/ql-vscode/src/vscode-tests/cli-integration/remote-queries/run-remote-query.test.ts

```diff
@@ -353,8 +328,8 @@ describe('Remote queries', function() {
     });
   });
 
-  function verifyQlPack(qlpackPath: string, queryPath: string, packVersion: string, pathSerializationBroken: boolean) {
-    const qlPack = yaml.load(fs.readFileSync(qlpackPath, 'utf8')) as QlPack;
+  function verifyQlPack(queryPath: string, contents: Buffer, packVersion: string, pathSerializationBroken: boolean) {
```

Comment:
Should `queryPath` be `queryPackPath`?

Reply:
I don't think so, this is not the path to the query pack, but the path to the query which is mentioned within the query pack. For example, it would be a YAML file like:

```yaml
name: codeql-remote/query
version: 0.0.0
dependencies:
   codeql/javascript-all: '*'
library: false
defaultSuite:
  - description: Query suite for variant analysis
  - query: in-pack.ql
```

The last `in-pack.ql` is the `queryPath`.

Reply:
Oh I see! Thanks!